mina.netty/src/main/java/org/kaazing/mina/core/service/AbstractIoService.java

```diff
@@ -306,7 +306,11 @@ public final void dispose() {
                 try {
                     this.disposalFuture = disposalFuture = dispose0();
```

Comment:
Can we identify which connector is returning null for dispose0, and fix that instead of doing the NPE guard?  We should then be able to test it